src/main/java/net/neoforged/neoforge/fluids/capability/templates/FluidTank.java

```diff
@@ -22,6 +26,11 @@ public class FluidTank implements IFluidHandler, IFluidTank {
     protected FluidStack fluid = FluidStack.EMPTY;
     protected int capacity;
 
+    public static final Codec<FluidTank> CODEC = RecordCodecBuilder.create(i -> i.group(
+        Codec.INT.fieldOf("capacity").forGetter(x -> x.capacity),
+        FluidStack.CODEC.optionalFieldOf("Fluid", FluidStack.EMPTY).forGetter(x -> x.fluid)
```

Comment:
Mojang has seemingly been moving towards using snake case for all fields in codecs, so I think the fluid field should as well, rather than being capitalized.